internal/core/exec.go

```diff
@@ -0,0 +1,28 @@
+package core
+
+import (
+	"context"
+	"os"
+	"os/exec"
+)
+
+type OverrideExecFunc func(cmd *exec.Cmd) (exitCode int, err error)
+
+func defaultOverrideExec(cmd *exec.Cmd) (exitCode int, err error) {
+	err = cmd.Run()
+	if exitErr, isExitErr := err.(*exec.ExitError); isExitErr {
+		return exitErr.ExitCode(), nil
+	}
+	return 0, err
+}
+
+func ExecCmd(ctx context.Context, cmd *exec.Cmd) (exitCode int, err error) {
+	meta := extractMeta(ctx)
+
+	// We do not support override of stdin
+	cmd.Stdout = os.Stdin
```

Comment:
We need a way to bring the stdin in the command (for instance registry login, the stdin is a io.Reader built from the secret key)

Reply:
Also it's `cmd.Stdin` :smile: